debugd/internal/debugd/logcollector/logcollector.go

```diff
@@ -99,6 +100,20 @@ func NewStartTrigger(ctx context.Context, wg *sync.WaitGroup, provider cloudprov
 				return
 			}
 
+			logger.Infof("Getting logstash config template")
+			tmpl, err = getFilebeatTemplate(ctx, logger)
+			if err != nil {
+				logger.Errorf("Getting filebeat config template: %v", err)
+				return
+			}
+			filebeatConf := filebeatConfInput{
+				LogstashHost: "localhost:5044",
+			}
+			if err := writeFilebeatConf(tmpl, filebeatConf); err != nil {
```

Comment:
can you include in the proposed changes why we add filebeat here?

Reply:
could you test that the changes in the debugd logcollector are working?

Reply:
> could you test that the changes in the debugd logcollector are working?

yes, doing so already

Reply:
> can you include in the proposed changes why we add filebeat here?

what do you mean?

Reply:
> > can you include in the proposed changes why we add filebeat here?
> 
> what do you mean?

nvm, saw that we deployed filebeat equally before